src/themes/nice/mixins/text.scss

```diff
@@ -0,0 +1,25 @@
+@import '../base-variables/typography';
+@import '../base-variables/colors';
+@import '../base-variables/sizing';
+
+@mixin text-font {
+    font-size: $font-size-normal;
+    line-height: $font-size-normal * 1.5;
+    color: $color-foreground;
+}
+
+@mixin text-link {
+    color: $color-primary;
+    text-decoration: $link-decoration-normal;
+
+    &:hover {
+        text-decoration: $link-decoration-hover;
+    }
+}
+
+@mixin heading-1 {
```

Comment:
What about other headings (h2, h3 etc.)? Maybe we can consider making this a more 'smart' mixin so that you can e.g. provide the heading size as a mixin argument, and have it return the properties accordingly?
e.g.
```
$heading-font-sizes: $h1-font-size, $h2-font-size, $h3-font-size, $h4-font-size;
@mixin heading($n: 1) {
    font-family: $heading-font-family;
    font-size: nth($heading-font-sizes, $n);
    color: $color-secondary;
    margin: $spacing-large 0;
}
```

Reply:
cool idea. Maybe we can have some more definitions for the headings. Maybe line-height and margins should change be relative to font?

Reply:
Yes maybe that will be good, let's fine tune that later on as you suggest.